src/coreclr/src/tools/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -2882,7 +2882,17 @@ private uint getJitFlags(ref CORJIT_FLAGS flags, uint sizeInBytes)
             }
 
             if (this.MethodBeingCompiled.IsNativeCallable)
+            {
+#if READYTORUN
+                if (targetArchitecture == TargetArchitecture.X86
+                    && _compilation.TypeSystemContext.Target.OperatingSystem == TargetOS.Windows)
+                {
+                    throw new NotSupportedException("ReadyToRun: Methods with NativeCallableAttribute not supported");
```

Comment:
NotImplementedException may be more appropriate for these, no sure.

Reply:
Oh. Did I misread your comment? They were originally `E_NOTIMPL` so that makes sense. I thought you said to make them `NotSupportedException`.

Reply:
Yep. I misread your comment.